### PR TITLE
fix: change the label

### PR DESCRIPTION
**Pull request by sunilsabatp**
to do-> change the label for` in-progress`  in` /profile/history` to `Pending`
image

### CODE REVIEW

File: src/features/user/Account/History.tsx [hunk 1]
```diff
@@ -151,6 +151,16 @@ export default function History({
...
                       </div>
+                    ) : (
+                      <div
+                        key={item[0]}
+                        className={`${styles.filterButton} ${
+                          filter === item[0] ? styles.selected : ''
+                        }`}
+                        onClick={() => handleSetFilter(item[0])}
+                      >
+                        {t('me:pending')}
```
Comment (norbertschuler) [inline src/features/user/Account/History.tsx:162]: You seem to have duplicated the whole div-tag looking like the same code, is it? Why don't you just make a decision on what translation resource to use, e.g. `{t(item[0] !== 'in-progress' ? item[0] : 'me:pending')}`?

**Review by norbertschuler — CHANGES_REQUESTED** [review]

File: src/features/user/Account/History.tsx (hunk 1 above)
Comment (norbertschuler) [reply src/features/user/Account/History.tsx:162]: Btw. this looks like quite a hack for the backend returning a wrong status, is it? We should add a comment about this for some future developer looking at it!

Comment (sunilsabatp) [reply src/features/user/Account/History.tsx:162]: it is going to be fix in the backend , as in frontend we are mapping the response to display  button

**Comment by sunilsabatp** [comment]
issue fixed in the backend

**Pull request closed without merging**